src/gui/dialogs/multiplayer/lobby.cpp

```diff
@@ -657,6 +658,15 @@ void mp_lobby::pre_show(window& window)
 		}
 	}
 
+	stacked_widget& pager = find_widget<stacked_widget>(&window, "tabs_container", false);
```

Comment:
What is this for still? Should be able to go.

Reply:
The default layer for a stacked widget is set to -1, which would prevent most interactions with the game list if not set to 0 instead.

The `VALIDATE` is there since that appeared to be the standard thing to do when using a stacked widget.

Reply:
A stacked widget's "layer" should really be a bit field, as its _intended_ purpose is to display multiple things on top of each other…

More to the point, and this isn't really relevant to the PR anymore, but… even when all the layers are visible, are you saying the stacked widget doesn't allow interactions even with the topmost layer?